pkg/mdns/server/server_test.go

```diff
@@ -1,49 +0,0 @@
-package server
-
-import (
-	"net"
-	"strings"
-	"testing"
-)
-
-func TestFunctionalServer(t *testing.T) {
-	// var stopCh = make(chan struct{})
```

Comment:
We commented out the contents of this test because it wasn't working, but we wanted to keep the file there so we could come back and fix the test later.

Reply:
Yes, this is a valid concern.

Could we use Jira or GutHub issues to keep track of our TODOs, so they don't get lost and get prioritized?

If you prefer to do that in the code base, I will add it again.

Reply:
I filed USHIFT-335 to keep track of the need to come back to those tests.